modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -73,7 +73,10 @@ For both cases, if a claim is present, the policy will validate the token value,
 
 == Claim Propagation
 
-After the JWT is parsed, the policy sets the claims in the authentication object in a map under the *claims* property.
+Once the policy parses and validates the JWT, the claims are propagated to downstream policies and flows through the *claims* property in the authentication object.
```

Comment:
After the policy parses and validates the JWT (use full form of JWT), the claims are propagated to downstream policies and flow, by using the *claims* property in the authentication object.